js/src/components/gallery-init.js

```diff
@@ -1,22 +1,20 @@
-/* global cloudinaryGalleryConfig */
+/* global CLD_GALLERY_CONFIG */
 
-const configElements = document.querySelectorAll(
-	'[data-cloudinary-gallery-config]'
-);
+( () => {
+	/**
+	 * A way to catch the galleryWidget function which comes from the `cloudinary` object.
+	 * The `cloudinary` object is later overwritten by the video player library by Cloudinary,
+	 * which effectively removes the gallery widget library from the page.
+	 */
+	const { galleryWidget } = cloudinary;
```

Comment:
Is this file going through a transpiler to be compiled for all browsers? Object destructuring isn't supported in IE, for example https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Destructuring_assignment#browser_compatibility

Reply:
Yes @kasparsd, Babel should be doing the transpile of the code. Also, we dropped support for IE :)